Makefile.am

```diff
@@ -103,10 +103,8 @@ condconfigexists = ConditionPathExists=\|/etc/sssd/sssd.conf\nConditionDirectory
 # Capabilities usage by binaries:
 #   - 'ldap_child': read keytab (dac_read_search)
 #   - 'krb5_child':
-#         - store TGT for a given user (set*id);
-#         - create path components of DIR:/FILE: cache, for example: /run/user/$UID (dac_override, chown)
```

Comment:
Who creates it now? /run/user/$UID is created by systemd, but what if the path is different?

Reply:
Not SSSD business - `kinit` doesn't either. See discussion in the team mail list.